nix-utils/hackage/default.nix

```diff
@@ -0,0 +1,64 @@
+{ reflex-platform ? import ../.. {} }:
```

Comment:
Can we make this explicit arguments that it needs from reflex-platform instead? That is be more specific in what parts of reflex-platform you actually need.